spec/lib/middleman-cdn/commands_spec.rb

```diff
@@ -15,6 +15,12 @@
         described_class.say_status(nil, "a status")
       end
     end
+
+    context "with a cdn" do 
+      it "should say" do
+        described_class.say_status("the cdn", "a status")
+      end
+    end
```

Comment:
This test doesn't really test much, neither does the test above it that I wrote either. Sorry about that. Leave it as it is at the moment, I'll try and improve on them as a whole post merge.